src/LexModelsV2/LexModelsV2Client.php

```diff
@@ -86,4 +88,37 @@
  * @method \Aws\Result updateSlotType(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateSlotTypeAsync(array $args = [])
  */
-class LexModelsV2Client extends AwsClient {}
+class LexModelsV2Client extends AwsClient {
+    public function __construct(array $args)
+    {
+        parent::__construct($args);
+
+        // Setup middleware.
+        $stack = $this->getHandlerList();
+        $stack->appendBuild($this->updateContentType(), 'models.lex.v2.updateContentType');
+    }
+
+    /**
+     * Creates a middleware that updates the Content-Type header when it is present
```

Comment:
May make sense to add a little more detail to justify why this is needed (just that service has a protocol of rest-json but the operations require it be set differently)